core/src/test/scala/integration/kafka/api/TransactionsTest.scala

```diff
@@ -157,6 +163,14 @@ class TransactionsTest extends KafkaServerTestHarness {
     readCommittedConsumer.assignment.asScala.foreach { tp =>
       assertEquals(1L, readCommittedConsumer.position(tp))
     }
+
+    // undecided timestamps should not be searchable either
+    val offsetsForTimes = readCommittedConsumer.offsetsForTimes(Map(
```

Comment:
The path is covered elsewhere, but I guess it doesn't hurt to make the expectation clear.